lightning/src/blinded_path/payment.rs

```diff
@@ -246,8 +246,16 @@ impl BlindedPaymentPath {
 		Self { inner_path, payinfo }
 	}
 
-	#[cfg(any(test, fuzzing))]
-	pub fn from_raw(
+	/// Builds a new [`BlindedPaymentPath`] from its constituent parts.
+	///
+	/// Useful when reconstructing a blinded path from previously serialized components.
+	///
+	/// Parameters:
+	/// * `introduction_node_id`: The public key of the introduction node in the path.
+	/// * `blinding_point`: The public key used for blinding the path.
+	/// * `blinded_hops`: The encrypted routing information for each hop in the path.
+	/// * `payinfo`: The [`BlindedPayInfo`] for the blinded path.
+	pub fn from_blinded_path_and_payinfo(
```

Comment:
This one could call into `from_parts`.

Or `BlindedPath` and `from_parts` could be made public instead and this method can be dropped?

Reply:
sure, the first one makes totally sense.